lib/good_job/active_job_extensions/logging.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+module GoodJob
+  module ActiveJobExtensions
+    module Logging
+      extend ActiveSupport::Concern
+
+      included do
+        around_perform do |job, block|
+          original_logger = job.logger
+          job.logger = ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(LogDevice.new(job)).extend(ActiveSupport::Logger.broadcast(original_logger)))
```

Comment:
@bensheldon , the `around_perform` hook above makes sense to me but what do L10 and L11 do exactly please ?

Reply:
This is creating a new logger that outputs to the custom LogDevice, and then teeing the output via `broadcast` so that the output still goes to the old logger too.

I could probably break this up into multiple lines. e.g.

```ruby
custom_logger = ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(LogDevice.new(job))
custom.logger.extend(ActiveSupport::Logger.broadcast(original_logger))

ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(LogDevice.new(job)).extend(ActiveSupport::Logger.broadcast(original_logger)))
```